trie/iterator.go

```diff
@@ -946,17 +946,14 @@ func newMerkleTreeIterator(
 	}
 	if it.err == nil && rootNode != nil {
 		it.stack = []merkleTreeIteratorNode{rootNode}
-		it.seek(start)
+		if len(start) != 0 {
+			it.seek(zk.NewTreePathFromHashBig(common.BytesToHash(start)))
+		}
 	}
 	return it
 }
 
-func (it *merkleTreeIterator) seek(path []byte) {
-	if len(path) == 0 {
-		return
-	}
-	path = zk.NewTreePathFromHashBig(common.BytesToHash(path))
-
+func (it *merkleTreeIterator) seek(path zk.TreePath) {
 	for _, p := range path {
 		if parent, ok := it.stack[len(it.stack)-1].(*merkleTreeIteratorParentNode); ok {
```

Comment:
I think you should check if `len(it.stack) > 0` before this line.

Reply:
Feedback on stack length 0: https://github.com/kroma-network/go-ethereum/pull/118#discussion_r1855684834